tutorial.md

```diff
@@ -0,0 +1,124 @@
+# Compositional differential abundance testing with ALDEx2 with Moving Pictures dataset
+
+ALDEx2 is a differential abundance package that was initially developed for meta-RNA-Seq, but has performed very well with traditional RNA-Seq, 16S rRNA gene sequencing, and selective growth-type (SELEX) experiments. In principle, ALDEx2 is generalizable and can be applied to nearly any type of data that is generated by high-throughput sequencing that generates tables of per-feature counts for each sample [Fernandes 2014](https://microbiomejournal.biomedcentral.com/articles/10.1186/2049-2618-2-15). Here we will apply it to the "Moving Pictures" dataset by [Caporaso et al. 2011](https://www.ncbi.nlm.nih.gov/pubmed/21624126) to investigate differential abundance of the gut in two individuals.
+
+We will start with the artifact generated by the DADA2 pipeline, `table.qza`. The metadata can be downloaded in the first step in the [Moving Pictures tutorial](https://docs.qiime2.org/2019.7/tutorials/moving-pictures/). The first step for ALDEx2 is to filter the samples to only compare the gut site.
+
+```
+qiime feature-table filter-samples \
+  --i-table table.qza \
+  --m-metadata-file sample-metadata.tsv \
+  --p-where "[body-site]='gut'" \
+  --o-filtered-table gut-table.qza
+```
+
+The next step will be to run ALDEx2. The full pipeline is implemented in the `aldex2` function The input is the `FeatureTable[Frequency]`, as well as a metadata file. The metadata file is necessary for defining the different groups you will be testing. For this tutorial, the groups are identified by the subject column from the metadata file. ALDEx2 automatically adds a pseudocount to remove zeros from the data, and filters any samples with 0 reads.
+
+```
+qiime aldex2 aldex2 --i-table gut-table.qza --m-metadata-file gut-metadata.tsv --p-condition subject --output-dir gut-test
```

Comment:
I'd recommend changing gut-metadata.tsv to sample-metadata.tsv so that it is consistent with the moving pictures tutorial.

Reply:
agreed!